### ISSUE TITLE
TIme for build

### ISSUE THREAD

**Issue by amanbawa1996** [NVIDIA/workbench-example-hybrid-rag#19]
This build at least takes 2 hours, not several minutes

**Comment by jtcasablanca** [comment]
Yeah. Container builds can vary quite a bit depending on the internet connection and the system you're building it on.

We will specify a range of times.

In order to get a better idea for your situation, what's the internet and system you are having this 2 hour build on?
